assets/js/blocks/cart-checkout/cart/test/block.js

```diff
@@ -13,7 +13,7 @@ import { defaultCartState } from '../../../../data/default-states';
 
 describe( 'Testing cart', () => {
 	beforeEach( async () => {
-		fetchMock.mockResponseOnce( ( req ) => {
+		fetchMock.mockResponse( ( req ) => {
```

Comment:
we need this so we don't end up mocking each request (causing multiple requests in the empty test for example).